frontend/web/components/StaleFlagWarning.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC } from 'react'
+import Tag from './tags/Tag'
+import Constants from 'common/constants'
+import moment from 'moment'
+import { Project, ProjectFlag } from 'common/types/responses'
+import { getProtectedTags } from 'common/utils/getProtectedTags'
+import { IonIcon } from '@ionic/react'
+import { close, warning } from 'ionicons/icons'
+import Tooltip from './Tooltip'
+import ProjectStore from 'common/stores/project-store'
+
+type StaleFlagWarningType = {
+  projectFlag: ProjectFlag
+}
+
+const StaleFlagWarning: FC<StaleFlagWarningType> = ({ projectFlag }) => {
+  const protectedTags = getProtectedTags(projectFlag, `${projectFlag.project}`)
+  if (protectedTags?.length) {
+    return null
+  }
+  const created_date = projectFlag?.created_date
+  const daysAgo = created_date && moment().diff(moment(created_date), 'days')
+  console.log(ProjectStore.model)
+  const suggestStale =
+    daysAgo >=
+    ((ProjectStore.model as Project | null)?.stale_flags_limit_days || 30)
```

Comment:
Hmm... I did consider whether we should be using this field, but I'm not 100% certain we should be. The reason being that (a) this field is only available to enterprise customers and (b) it is used by the stale flags tagging logic as a comparison on when a feature was last changed in a given environment. 

It's probably fine for now, since it will be 30 days for everyone, and maybe we should create another field to hold the data for this.

Reply:
Hmm yeah I also deliberated this quite a bit, I wonder if the duration should be longer than stale flags too.

Reply:
Yeah, I think so

Reply:
But agree we can leave for now!